v7/nanogallery_directive/templates/mako/embedded-nanogallery.tmpl

```diff
@@ -0,0 +1,51 @@
+## -*- coding: utf-8 -*-
```

Comment:
write a Jinja version (the jinjify script could help you here, if you get it to work for one file)